sdk/js/src/EventQueue.ts

```diff
@@ -1,5 +1,5 @@
-import { DVCClient } from './Client'
-import { DVCEvent } from './types'
+import { DevCycleClient } from './Client'
+import { DevCycleClientEvent } from './types'
```

Comment:
why client event and not just "event"? what is a "client"?

Reply:
yea `DevCycleEvent` makes more sense.